tests/loadtests.py

```diff
@@ -74,23 +82,28 @@ async def test_basic_topic(session):
         await ws.send_json(dict(messageType="register", channelID=channel_id))
         msg1 = await ws.receive_json()
         await ws.close()
-
+    # switch to rust endpoint
+    if os.getenv("AUTOPUSH_RUST_SERVER"):
+        path = urlparse(msg1["pushEndpoint"]).path
+        endpoint_url = urljoin(URLS["rs_server_url"], path)
+    else:
+        endpoint_url = msg1["pushEndpoint"]
     # Send Notification
     async with session.post(
-        msg1["pushEndpoint"],
+        endpoint_url,
         headers=headers,
         data=base64.urlsafe_b64decode(encrypted_data[0]),
         ssl=False,
     ) as conn:
-        assert conn.status == 201
+        assert conn.status == 201 or 202

```

Comment:
We might want to revisit this in the future.

There was [an issue](https://github.com/mozilla-services/autopush/issues/1443) filed on Autopush a while ago noting that the RFC specification says to return 201 unless there's a receipt associated (we ignore receipt requests so basically, we shouldn't return 202). This changed the "legacy" mode where we returned 202 if a subscription was stored instead of delivered, which, in retrospect, was a terrible idea anyway.

This change is still percolating though distribution (including on the autopush-rs side).

Reply:
Sounds good. I will file an issue.